package.json

```diff
@@ -16,9 +18,23 @@
   },
   "homepage": "https://github.com/zhoutwo/url_connect#readme",
   "devDependencies": {
+    "babel": "^6.23.0",
+    "babel-core": "^6.24.1",
+    "babel-loader": "^6.4.1",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "eslint": "^3.19.0",
+    "eslint-config-google": "^0.7.1",
+    "eslint-plugin-react": "^6.10.3",
     "gulp": "github:gulpjs/gulp#4.0",
     "gulp-cli": "github:gulpjs/gulp-cli",
+    "prop-types": "^15.5.8",
+    "react": "^15.5.4",
```

Comment:
Even after compilation, React's code will be run. This is different from TypeScript stuff, for example